expiringdict/__init__.py

```diff
@@ -110,7 +113,7 @@ def get(self, key, default=None, with_age=False):
     def items(self):
         """ Return a copy of the dictionary's list of (key, value) pairs. """
         r = []
-        for key in self:
+        for key in list(self.keys()):
```

Comment:
I am not familiar with python 3.x, why do we need `list` here? Anyway even if it is needed for 3.x it is definitely not needed in 2.x and it would be better not to have this overhead when it is not needed.